lib/pbench/server/database/alembic.run

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# Alembic command expects to run in our database directory.
+cd lib/pbench/server/database
+
+# Our env requires a pbench-server config file.
+export _PBENCH_SERVER_CONFIG=$(pwd)/pbench-server.cfg
+
+# Had to run the command below before I could create the initial tables.
+#alembic stamp head
+
+# Then I could create the initial migration against an empty database.
+#alembic revision --autogenerate -m "Initial tables"
```

Comment:
It seems very weird to add a script to our repo in which all of the functional lines are commented out.  Short of explicitly invoking `tox -e alembic`, does anything actually execute this script?  (And, does anything other than a human user invoke `tox` like that?)

If not, why not commit this script the way it is supposed to be run?  (Otherwise, I think we should omit it from the PR...which, opens the question of what to do with the `tox.ini` change....)

Reply:
I removed this file entirely.